html/download-from-appProperty-in-server-xml.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<!--
+  Copyright (c) 2020 IBM Corporation and others.
+  All rights reserved. This program and the accompanying materials
+  are made available under the terms of the Eclipse Public License v1.0
+  which accompanies this distribution, and is available at
+  http://www.eclipse.org/legal/epl-v10.html
+ 
+  Contributors:
+      IBM Corporation - initial API and implementation
+-->
+<html style="height: 100%;">
+<head>
```

Comment:
add <title> to the template

Reply:
This is a DAP issue and not relevant to the subject of this issue.  It should be addressed in https://github.com/OpenLiberty/iguide-microprofile-config-intro/issues/254.